src/Extensions/ExtensionManager.php

```diff
@@ -13,18 +14,34 @@ public function getPackageManager()
 
     public function install($extensionId, $version = null)
     {
+        // @TODO temporary work around to increase memory
+        ini_set('memory_limit', '2gb');
```

Comment:
i had 512mb on my local dev and it kept complaining.. maybe we can make it configurable because some hosts might allow increasing memory temporarily but not give access to a php.ini